### PR TITLE
Increase build speed

### PR DESCRIPTION
**Pull request by RobbinHabermehl**
Stage view:
https://ci.datapunt.amsterdam.nl/job/Atlas/job/feature%252Ftg-2550-update-node/

Console output:
https://ci.datapunt.amsterdam.nl/job/Atlas/job/feature%252Ftg-2550-update-node/5/consoleFull

Goals:
[x] Increase build speed
[x] Assure no loss in testing

Additional performance gains (do so in another PR):
- create custom Node/Java Docker image;
- [cache Node dependencies](https://github.com/DatapuntAmsterdam/atlas/pull/717#issuecomment-314373497)

### CODE REVIEW

**Comment by RRMoelker** [comment]
De gemiddelde build kan een stuk sneller door package install step in de Dockerfile los te zetten van de rest van de build. Zie: https://github.com/DatapuntAmsterdam/mijn-amsterdam/blob/master/Dockerfile#L13

Ik neem aan dat de images gebuild worden met cache, in dat geval dan is het niet elke keer nodig om alle packages te downloaden.

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

**Comment by RobbinHabermehl** [comment]
test this please

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
   },
   "scripts": {
     "preinstall": "npm cache verify",
-    "postinstall": "./node_modules/protractor/node_modules/webdriver-manager/bin/webdriver-manager update && bower cache clean && bower install",
+    "postinstall": "./node_modules/protractor/node_modules/webdriver-manager/bin/webdriver-manager update",
```
Comment (boris-arkenaar) [inline package.json]: `bower install` niet nodig?

Comment (RobbinHabermehl) [inline package.json]: Zit in de `Dockerfile` nu!

**Pull request merged**